backend/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+authors = ["Mark-Simulacrum <mark.simulacrum@gmail.com>"]
+name = "rs-backend"
+version = "0.1.0"
+
+[dependencies]
+chrono = "0.2.22"
+error-chain = "0.2.2"
+json = "0.10.1"
+iron = "0.4.0"
+router = "0.2.0"
+persistent = "0.2.0"
+
+[profile.release]
+debug = true
```

Comment:
Well, until we optimize the `ServerData::from_fs` function it's very slow (55 seconds to read the `data/processed` in) to run it in debug mode, so I'm thinking it may be necessary for discovering where errors occurred in production. I'm not sure what the "correct" way to do this is though (I've written little Rust code with proper error handling before now).